From 5292545265af6ae4e8140b1af89369f7ae9be554 Mon Sep 17 00:00:00 2001
From: "Jiri (George) Lebl" <jiri.lebl@gmail.com>
Date: Thu, 3 Apr 2014 08:02:02 -0500
Subject: Thu Apr 03 08:01:49 2014  Jiri (George) Lebl <jirka@5z.com>

	* src/inter.c, src/genius-readline-helper.c: apply patch from Felipe
	  Sateler of debian to fix build with new readline.

diff --git a/src/genius-readline-helper.c b/src/genius-readline-helper.c
index 7e9e84b..dbf8fd2 100644
--- a/src/genius-readline-helper.c
+++ b/src/genius-readline-helper.c
@@ -183,7 +183,7 @@ main(int argc, char *argv[])
 	rl_terminal_name = "xterm";
 	rl_readline_name = "Genius";
 	rl_attempted_completion_function =
-		(CPPFunction *)tab_completion;
+		(rl_completion_func_t *)tab_completion;
 
 	while(fgets(buf,4096,infp)) {
 		int count;
diff --git a/src/inter.c b/src/inter.c
index 915f115..f0d2558 100644
--- a/src/inter.c
+++ b/src/inter.c
@@ -367,5 +367,5 @@ init_inter(void)
 {
 	rl_readline_name = "Genius";
 	rl_attempted_completion_function =
-		(CPPFunction *)tab_completion;
+		(rl_completion_func_t *)tab_completion;
 }
-- 
cgit v0.10.1

